package-lock.json

```diff
@@ -1571,6 +1571,315 @@
         "to-fast-properties": "^2.0.0"
       }
     },
+    "@firebase/analytics": {
+      "version": "0.6.9",
+      "resolved": "https://registry.npmjs.org/@firebase/analytics/-/analytics-0.6.9.tgz",
+      "integrity": "sha512-G0PkfMq/4tpDXwk/S2LKrXUWiz5tpQ6o2Lf6esgdEcDLpimPl32TrioNkDEDz8Xp0mzpY04UKwvYjT5xuzoKug==",
+      "requires": {
+        "@firebase/analytics-types": "0.4.0",
+        "@firebase/component": "0.4.1",
+        "@firebase/installations": "0.4.25",
```

Comment:
this should not be committed

Reply:
### Frontend work to be done
- [ ] **Home section**
- [ ] **Form section**
- [ ] **About section**
- [ ] **Prediction section**